tools/mo/unit_tests/mo/back/moc_preprocessing_test_actual.py

```diff
@@ -238,6 +238,32 @@ def test_mean_scale_with_layout(self):
         # Verify that layout presents in function after preprocessing
         self.assertEqual(function.get_parameters()[1].layout, Layout("NHWC"))
 
+    def test_mean_scale_with_layout_no_tensor_names(self):
+        argv = Namespace(mean_scale_values={'input2': {'mean': np.array([1., 2., 3., 4.]),
```

Comment:
what if first input is "input2" without tensor names, and second input has tensor name "input2:0" ?

Reply:
We will have two same tensor names, but I don't think this can happen in a real model. Since this is a workaround it shouldn't be needed if the bug with tensor names is fixed.